code/main.cpp

```diff
@@ -7,13 +7,12 @@ int main(int argc, char *argv[])
     QApplication app(argc, argv);
     MainWindow *w = new MainWindow();
 
-    Logger *logger = new Logger();
-    w->show();
+    Logger::SetStream(w->GetLogView());
+    Logger::Log("Logger initialized.");
+
+    Logger::Log("ERROR: No controllers were found!", Logger::ERROR);

```

Comment:
Yeap, I will be taking it out, probably move both the statements into their corresponding initializers/ctors.